src/library/browse/browsetablemodel.h

```diff
@@ -37,35 +37,42 @@ const int COLUMN_REPLAYGAIN = 20;
 // The BrowseTable models displays tracks
 // of given directory on the HDD.
 // Usage: Recording and Browse feature.
+//
+// NOTE(uklotzde): Accessing tracks from the browse view will
```

Comment:
This was intentional at the time since some people use Browse instead of the library and we want to let them edit the tracks in a way that persists across sessions and we didn't want to edit the files on disk by default unless the user opts in to that.